docs/setup/configuring-logging.asciidoc

```diff
@@ -0,0 +1,697 @@
+[[logging-configuration]]
+== Configure logging
```

Comment:
This page serves as a more detailed guide about the logging system and goes into the details we used to have in the dev docs section.